module-4/backend/week_three.md

```diff
@@ -5,15 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is Node?
+Node is a framework for developing serverside javascript. It will allow you to run javascript without a client. 
 2. What is Express? What is Express similar to in the Ruby world?
+Express is a backened framework that will allow you to create models and controllers. It is analogous to Sinatra in the ruby world.  
```

Comment:
Yes though we can created models and controllers without Express. Express itself is pretty un-opinionated and we choose to set up our project in an MVC format. Express allows us to create an api easier than pure javacript with Node